hack/rpm-deps.sh

```diff
@@ -20,6 +20,17 @@ virt_v2v="
   virtio-win
 "
 
+ovirt-imageio="
+  gcc
+  py3-pip
```

Comment:
```suggestion
  python3-pip
```

Reply:
I think that's the name of the package in alpine, perhaps we should use some UBI go tools image instead